### ISSUE TITLE
setEnableInitialSmsButton was removed in androidsdk 5.2.0

### ISSUE THREAD

**Issue by huynhquangtoan** [underscopeio/react-native-facebook-account-kit#199]
**setEnableInitialSmsButton** was removed in androidsdk 5.2.0

> /node_modules/react-native-facebook-account-kit/android/src/main/java/io/underscope/react/fbak/RNAccountKitModule.java:242: error: cannot find symbol
        configurationBuilder.setEnableInitialSmsButton(setEnableInitialSmsButton);

**Solution**:

> Add accountKitSdkVersion = '5.0.0' in root project's build.gradle

* React Native Account Kit version: 2.0.0
* React Native version: 0.60.4
* Platform(s) (iOS, Android, or both?): Android

**Comment by yosukapro** [comment]
Same issue

**Comment by jpgarcia** [comment]
I'm closing this as **v2.0.0** already sets the default version for the SDK  to **5.+**(https://github.com/underscopeio/react-native-facebook-account-kit/blob/master/android/build.gradle#L8)

You may face this issue if you previously specified **accountKitSdkVersion** on your project or maybe it could be related to a gradle cache issue.

**Comment by huynhquangtoan** [comment]
> I'm closing this as **v2.0.0** already sets the default version for the SDK to **5.+**(https://github.com/underscopeio/react-native-facebook-account-kit/blob/master/android/build.gradle#L8)
> 
> You may face this issue if you previously specified **accountKitSdkVersion** on your project or maybe it could be related to a gradle cache issue.

**setEnableInitialSmsButton has been removed in the latest version (5.2.0)**

Please check this: https://developers.facebook.com/docs/reference/androidsdk/current/AccountKit/

**SDK 5.0.0:**
https://developers.facebook.com/docs/reference/androidsdk/5.0.0/AccountKit/

**Comment by jpgarcia** [comment]
Got it! you are right. I'll reopen this then

Thanks for the details!

**Comment by kevinkevw** [comment]
My temp workaround is adding this into build.gradle 

`buildscript {
    ext {
         ...
         accountKitSdkVersion = "5.0.0"
    }
}
`

**Comment by anhtuank7c** [comment]
Both Android and iOS needed to limited AccountKit library version `5.0.0`, I faced the same issue on iOS after `pod repo update`, account kit `v5.3.0` doesn't compatible

**Comment by jpgarcia** [comment]
Guys, I've just published a new **2.1.0** version which solves the issues you reported here.

You should now be able to remove the workarounds on your `Podfile` and `build.gradle` files.

Before running the project make sure you refresh all the dependencies by doing this:

```bash
rm -rf ./node_modules ./ios/Pods ./ios/Podfile.lock
cd android
./gradlew clean
cd ../
yarn
```
Let me know if you found any issue.
